### ISSUE TITLE
missing css

### ISSUE THREAD

**Issue by shangcloud** [theKashey/used-styles#43]
we use this library in our project, but find missing some style, the css selector is like this:
```css
.downloads-1u5ev.downloadsFallback-1btP7 .logo-2Dv5-,.downloads-1u5ev.downloadsV5-3wG0y .logo-2Dv5- {
    margin-left: .08rem;
    position: relative;
    width: .5rem;
    height: .5rem
}
```
```html
<div class="downloads-1u5ev downloadsV5-3wG0y"><svg viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" class="logo-2Dv5-" role="img" fill="#fb7701" stroke="none" stroke-width="18.962962962962962" preserveAspectRatio="none"></svg></div>
```
could you fix this problem?

**Comment by theKashey** [comment]
Thank you for the issue. It was a false-negative regression introduced during [critical CSS reduction](https://github.com/theKashey/used-styles/issues/30). Basically, classes nested on "double parents" were not supported at all, and there was no test ensuring the correct behavior.

Give a shot to `used-styles@2.4.2`
